airflow/jobs/triggerer_job.py

```diff
@@ -614,7 +614,8 @@ def set_trigger_logging_metadata(self, ti: TaskInstance, trigger_id, trigger):
 
         :meta private:
         """
-        ti.is_trigger_log_context = True
+        if ti:  # can be None in tests
+            ti.is_trigger_log_context = True
```

Comment:
Which test(s)? We may want to fix them instead

Reply:
Ideally yes but it was a good number of them like 8 or 9 I think. I would like to leave that for another day. Maybe a to-do would suffice?

Reply:
Yeah let’s do it in a follow-up PR.